source/unified-test-format/tests/invalid/clientEncryption-missing_client.yml

```diff
@@ -0,0 +1,15 @@
+description: clientEncryption-missing_client
```

Comment:
This test is redundant in light of `clientEncryption-missing_keyVaultClient`, which I proposed renaming to `clientEncryptionOpts-keyVaultClient-required`.

Reply:
Forgot to remove this test when reverting `client` -> `keyVaultClient`. Will remove.